### ISSUE TITLE
A transpiler question during jspm should be re-worded

### ISSUE THREAD

**Issue by yfain** [jspm/jspm.io#33]
While running jspm init the user gets the following question:

Do you wish to use an ES6 transpiler? [yes]:

Answering "no" results in not even showing the following question:
Which ES6 transpiler would you like to use, Traceur, Babel or TypeScript?

This sounds confusing and may the users believe that transpiling into ES6 syntax is the only option.
I'd rephrase these two questions as follows:

Do you wish to use a transpiler? [yes]:
Which transpiler would you like to use, Traceur, Babel or TypeScript?

**Comment by guybedford** [comment]
Good point, thanks I've clarified in https://github.com/jspm/jspm-cli/commit/03714c0654607da1ff2ab202eae1e118ac971790.

**Comment by yfain** [comment]
Thanks, it was quick :)
You've removed ES6 only from the first question. What about removing ES6 from the second one:
Which transpiler would you like to use, Traceur, Babel or TypeScript?

**Comment by guybedford** [comment]
We should still encourage that transpilers are seen as supersets of ES6. The idea was never to encourage obscure syntax running through this mechanism, only that which would be supported in browser. Unfortunately those flood gates have long since been past, so when we deprecate transpilers for plugins things will get simpler again :)
